Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-sorting/ContestSorting.tsx

```diff
@@ -69,7 +71,7 @@ const ContestSorting = ({ onSortClick }: IContestSortingProps) => {
                         Sorting
                     </Heading>
                     <List
-                      values={possibleSortingTypes}
+                      values={possibleSortingTypes.filter((sort) => sort.name !== 'OrderBy')}
```

Comment:
This is no longer needed if Sorting is hidden?

Reply:
True, i though it was better to keep it and have to logic for hiding the button but it might lead to difficulties understanding the behavior, so i will revert that change.